### PR TITLE
Migration: docker image

### PR DESCRIPTION
**Pull request by bilalesi**
create a new docker image for testing migration in dev cluster

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added necessary unit and integration tests.
- [ ] I have added screenshots (if applicable), in the comment section.

### CODE REVIEW

File: .github/workflows/migrate.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+on:
+  push:
+    branches:
+      - fusion-react-18-vite
+
+jobs:
+  publish-preview:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
```
Comment (Dinika) [inline .github/workflows/migrate.yml:11]: Is there a reason why we've added the timeout. Nothing wrong with, just curious.

File: src/shared/organisms/DataPanel/DataPanel.spec.ts [hunk 2]
```diff
@@ -1,14 +1,13 @@
-import { vi } from 'vitest';
+import { vi as jest } from 'vitest';
```
Comment (Dinika) [inline src/shared/organisms/DataPanel/DataPanel.spec.ts:1]: why this change? Maybe it's better to rename the occurences of the word `jest` underneath to `vitest`.

**Review by Dinika — CHANGES_REQUESTED** [review]

File: .github/workflows/migrate.yml (hunk 1 above)
Comment (bilalesi) [reply .github/workflows/migrate.yml:11]: this is was the same configuration in preview, i did not may attention but it won't make any issue

**Review by Dinika — APPROVED** [review]

File: src/shared/organisms/DataPanel/DataPanel.spec.ts (hunk 2 above)
Comment (bilalesi) [reply src/shared/organisms/DataPanel/DataPanel.spec.ts:1]: you are right about it i fixed it, in the next PR
